docs/wallet_API.md

```diff
@@ -6,27 +6,27 @@ author: Claude Barde
 
 ## What is the Wallet API?
 
-The Tezos blockchain is a fantastic tool, but it is self-contained. Except for the transactions you send to it, it has no interaction with the outside world. However, it would be amazing to interact with it, such as making payments, reading the balance of an account, or recording data in a smart contract. This interaction is what the Wallet API (and Taquito in general) achieves. The Wallet API provides a new yet familiar way to interact with the blockchain and smart contracts by delegating several actions that Taquito previously handled to the wallets. This delegation offers more flexibility for both developers and users and gives the ecosystem more space to evolve.
+You have learned how to use Taquito to interact with the Tezos blockchain. Up to this document, you have been using a signer to sign operations. For interactive dApps, it is common to use a wallet to sign operations. From a user's perspective, the workflow is as follows:
+
+1. The user has a wallet installed and configured on their device.
+2. The user visits a dApp and wants to interact with it.
+3. The dApp asks the user to connect their wallet.
+4. The user selects their wallet.
+5. The wallet asks the user to confirm the connection.
+6. Now when the dApp wants to send an operation, it asks the wallet to sign it. This might need additional confirmation from the user, or the user might have setup their wallet to sign operations automatically (for instance, automatically sign transactions below a certain amount in a certain time frame).
```

Comment:
[preference] think this line is beyond the current scope of wallet api introduction
> or the user might have setup their wallet to sign operations automatically (for instance, automatically sign transactions below a certain amount in a certain time frame).

Reply:
Agree. Still think here is a good chance to let the reader know about this possibility.

Reply:
If we don't have documentation on how to do that, I think we shouldn't mention it in the docs. Which I think is part of why An thinks it is out of scope

Reply:
We don't have documentation for that, because it is in the responsibility of the wallet. I included it here, because I believe it adds value to the big picture.
But I will remove it because you both prefer it to be removed.